src/main/java/no/uio/ifi/localega/doa/services/StreamingService.java

```diff
@@ -97,7 +97,7 @@ private InputStream getPlaintextResponse(byte[] header, InputStream bodyInputStr
         ByteArrayInputStream headerInputStream = new ByteArrayInputStream(header);
         SequenceInputStream sequenceInputStream = new SequenceInputStream(headerInputStream, bodyInputStream);
         Crypt4GHInputStream crypt4GHInputStream;
-        if (!StringUtils.isEmpty(startCoordinate) && !StringUtils.isEmpty(endCoordinate)) {
+        if (StringUtils.hasLength(startCoordinate) && StringUtils.hasLength(endCoordinate)) {
```

Comment:
While this is fine, we should probably just use `str.isBlank()` or `str.isEmpty()` instead depending on the use case.